sdk/communication/communication-chat/CHANGELOG.md

```diff
@@ -1,9 +1,11 @@
 # Release History
 
-## 1.6.0 (upcoming)
 
-### Features Added
+## 1.6.0-beta.2 (upcoming)
 
+### Features Added
```

Comment:
Hi @jimchou-dev, I just noticed that some previous beta version like 1.4.0-beta.1 Changelog update were lost during the merge from 1.6.0-beta.1. Could you please add them back?

Reply:
Added missing changelog items